src/cobra/core/group.py

```diff
@@ -1,9 +1,6 @@
-# -*- coding: utf-8 -*-
-
 """Define the group class."""
 
-from __future__ import absolute_import
-
+from typing import Iterable, Optional
 from warnings import warn
 
 from cobra.core.dictlist import DictList
```

Comment:
Please make the import as `from .dictlist import DictList` . Also, you can use this "relative import" for other cases as well.